### ISSUE TITLE
Clarification on the training/evaluation pipeline shown in the notebook

### ISSUE THREAD

**Issue by Ivan-Fan** [SalesforceAIResearch/gift-eval#25]
Hi! Really thanks for the splendid work!

Currently I am exploring time-series forecasting area and very happy to see a nice benchmark with a well-explained tutorial~ I am currently looking at the notebook [naive.ipynb](https://github.com/SalesforceAIResearch/gift-eval/blob/main/notebooks/naive.ipynb) and [feedforward.ipynb](https://github.com/SalesforceAIResearch/gift-eval/blob/main/notebooks/feedforward.ipynb) and there are some naive questions:) hope they do not bother~

1. In the formal evaluation pipeline, will statisticalforecast model use training data? I noticed only `dataset.test_data` is parsed into `evaluate_model` function, and there is no other data put in (maybe I missed sth.), while the feedforward model has an explicit training process.
2. For the feedforward model, it seems that it only trains on the validation set given `predictor = estimator.train(dataset.validation_dataset)`. Is this intended to save training time in this tutorial?

Thank you so much!

**Comment by cuthalionn** [comment]
Hi @Ivan-Fan, 

1. Statistical models don’t use any separate training data—they rely solely on the context within the test series to make forecasts. In contrast, deep learning models are trained specifically for each test series. That’s why some researchers refer to them as “specific models.” 

2. Regarding the validation set: in time series forecasting, the train/val/test split is done along the time axis rather than across data IDs. So the validation set includes the training portion of the time series. For each deep learning model we do the hyperparameter search using train-validation set. Then perform the final training on the validation set using the optimal hyperparameters.

Hope this clears things up! Let me know if you have any more questions.
